Makefile.coverity

```diff
@@ -0,0 +1,2 @@
+all:
+	make -C boot/coverity
```

Comment:
Does:
```
%:
        make -C boot/coverity $@
```
work. It'd be nice to pass through whatever build target was requested.

Reply:
Yes, I'm not sure what it would be useful for at the moment but there's no problem with adding it.